noir-projects/aztec-nr/aztec/src/utils/point.nr

```diff
@@ -9,6 +9,8 @@ global BN254_FR_MODULUS_DIV_2: Field = 10944121435919637611123202872628637544274
 /// We don't serialize the point at infinity flag because this function is used in situations where we do not want
 /// to waste the extra byte (encrypted log).
 pub fn point_to_bytes(pk: Point) -> [u8; 32] {
```

Comment:
Sneaked this comment in as it was on my mind for a while.

Reply:
sneaky sneaky